### ISSUE TITLE
error: Cannot read property 'Parameter' of undefined

### ISSUE THREAD

**Issue by murdercdh** [fsbahman/apidoc-swagger#61]
Now I can use apidoc to generate the html correctly.

But when I try to use command line to gen the swagger-ui json file like following:

`apidoc-swagger -i router/ -o swagger_doc/`

then I get the error like this :

'error: Cannot read property 'Parameter' of undefined'

It seems I have to debug from the apidoc-swagger source code?

**Comment by murdercdh** [comment]
![image](https://user-images.githubusercontent.com/5146156/98771589-c6090c80-241f-11eb-88f7-ca14a657339b.png)
then I debug a litte bit more from source code.
I find the line 274, if the fields is None, the "Parameter" will case the crash. so I change a litte. 
`	if (verbs.parameter &&(verbs.parameter.fields && verbs.parameter.fields.Parameter)) {
`
but it seems no one can do the code merge job right now?

**Comment by murdercdh** [comment]
link to https://github.com/fsbahman/apidoc-swagger/pull/57/commits/5fe4dd9ddda8b46c6c06d0b78e832ebfc03537bb

**Comment by skywangc** [comment]
Did you solve the problem by modifying the source code?

**Comment by murdercdh** [comment]
> Did you solve the problem by modifying the source code?

yes, I think you can merge that pull request in to you main branch.
